validator/client/validator_propose.go

```diff
@@ -126,8 +123,14 @@ func (v *validator) ProposeBlock(ctx context.Context, slot uint64) {
 	// 5. Broadcast to the network via beacon chain node.
 	blkResp, err := v.proposerClient.ProposeBlock(ctx, block)
 	if err != nil {
-		log.WithField("error", err).Error("Failed to propose block")
+		log.WithError(err).Error("Failed to propose block")
 		return
 	}
-	log.WithField("hash", fmt.Sprintf("%#x", blkResp.BlockHash)).Info("Proposed new beacon block")
+	log.WithFields(logrus.Fields{
+		"blockRoot": fmt.Sprintf("%#x", blkResp.BlockRootHash32),
+	}).Info("Proposed new beacon block")
+	log.WithFields(logrus.Fields{
```

Comment:
Can these two logs be combined?

Reply:
It makes the line very verbose and unreadable given we're logging a hash. It looks much nicer split this way in the runtime.